include/ppx/metrics.h

```diff
@@ -285,9 +386,24 @@ class Manager final
     // Returns whether a run is active.
     bool HasActiveRun() const;
 
+    // Allocate a MetricID.
+    MetricID AllocateID() { return EnsureAllocateID(kInvalidMetricID); }
```

Comment:
Is that needed?
Could this be removed, and only `AddMetric` and `AddLiveMetric` used instead?
To bind 2 metrics, the user would call one function first, and used the returned ID to create the second one.
This way, there is no way for an ID to bound to no metrics.

Reply:
Correct myself, it used to be that you need to re-AddMetric, which creates new metricID per run.